Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR.materialtype

```diff
@@ -0,0 +1,75 @@
+{
+    "description": "Material Type with properties used to define Base PBR, a metallic-roughness Physically-Based Rendering (PBR) material shading model.",
+    "version": 1,
+    "versionUpdates": [
+        {
+            "toVersion": 1,
+            "actions": [
+                {"op": "rename", "from": "irradiance.color", "to": "irradiance.manualColor"},
+                {"op": "setValue", "name": "irradiance.irradianceColorSource", "value": "Manual"}
+            ]
+        }
+    ],
+    "propertyLayout": {
+        "propertyGroups": [
+            {
+                "$import": "../MaterialInputs/BaseColorPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/MetallicPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/RoughnessPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/SpecularPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/NormalPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/UvPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/IrradiancePropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/GeneralCommonPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/OcclusionPropertyGroup.json"
+            },
+            {
+                "$import": "../MaterialInputs/EmissivePropertyGroup.json"
+            }
+        ]
+    },
+    "shaders": [
+        {
+            "file": "./NewPBR_ForwardPass.shader",
```

Comment:
I thought we talked about keeping alpha cutout in the standard material type, since it goes in the opaque pass. Or are you thinking to move that because of the EDS difference?

BTW, we were saying a while ago that we might not need a separate shader for EDS, the "[earlydepthstencil]" attribute was only required because we weren't using shader variants correctly. The compiler normally enables EDS automatically but disables it when it finds clip instructions in the pixel function. If we were using the right shader variant then the clip instruction would be dead-code-eliminated and presumably EDS would be enabled automatically for that variant. I don't know if we confirmed this, but think we can add alpha clipping here without complicating the shader setup.

Reply:
wouldn't EDS be faster when can be used though?

Reply:
Yes, but my point described above is that it should happen automatically if we set up the shader variants correctly.